README.md

```diff
@@ -38,12 +38,27 @@ There are currently two different metrics that you can set a maximum bound on:
     deleting the `TestResults` directory or any of its contents between
     runs.
 
-4. `--max-resource-stddev N`
+4. `--max-duration-cv N`
```

Comment:
Hate to make these names more cryptic, but I realized we should really have the "unit" attached as well, just like with `--max-duration-seconds` etc. So how about `--max-duration-cv-percent` or `--max-duration-cv-pct`? :)

In fact technically we should have had `--max-duration-seconds-stddev`, but I don't think that's worth the breaking change to fix.

Reply:
Good point. I'll change that.